Older clang-format can break the build

```
$ clang-format --version
LLVM (http://llvm.org/):
  LLVM version 3.4svn
  Optimized build.
  Built Dec 18 2013 (23:49:33).
  Default target: x86_64-apple-darwin13.4.0
  Host CPU: x86-64

$ make format
mkdir -p build/darwin
cd build/darwin && cmake ../.. && make --no-print-directory format
-- Boost version: 1.56.0
-- Thrift version 0.9.1
-- Found readline library
-- Found RocksDB
-- Found components for DL
-- Could NOT find Doxygen (missing:  DOXYGEN_EXECUTABLE)
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/zwass/osquery/build/darwin
Formatting code with clang-format
YAML:38:17: error: unknown key 'SpacesInAngles'
SpacesInAngles: false
                ^~~~~
Error reading /Users/zwass/osquery/.clang-format: Invalid argument
Can't find usable .clang-format, using LLVM style
YAML:38:17: error: unknown key 'SpacesInAngles'
SpacesInAngles: false
                ^~~~~
Error reading /Users/zwass/osquery/.clang-format: Invalid argument
Can't find usable .clang-format, using LLVM style
YAML:38:17: error: unknown key 'SpacesInAngles'
SpacesInAngles: false
                ^~~~~
... many more of same error

$ make
mkdir -p build/darwin
cd build/darwin && cmake ../.. && make --no-print-directory
-- Boost version: 1.56.0
-- Thrift version 0.9.1
-- Found readline library
-- Found RocksDB
-- Found components for DL
-- Could NOT find Doxygen (missing:  DOXYGEN_EXECUTABLE)
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/zwass/osquery/build/darwin
[  1%] Built target gtest
[  1%] Built target gtest_main
[  2%] Built target osquery_sqlite
Scanning dependencies of target osquery_tables
[  3%] Building CXX object osquery/tables/CMakeFiles/osquery_tables.dir/system/bash_history.cpp.o
[  5%] Built target osquery_tables
Scanning dependencies of target osquery_config
[  5%] Building CXX object osquery/config/CMakeFiles/osquery_config.dir/config.cpp.o
[  6%] Building CXX object osquery/config/CMakeFiles/osquery_config.dir/plugins/filesystem.cpp.o
[  7%] Building CXX object osquery/config/CMakeFiles/osquery_config.dir/plugins/glog.cpp.o
[  7%] Built target osquery_config
[ 22%] Built target osquery_generated_tables_depends
Scanning dependencies of target osquery_generated_tables
[ 22%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/bash_history.cpp.o
[ 23%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/etc_hosts.cpp.o
[ 24%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/groups.cpp.o
[ 24%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/last.cpp.o
[ 25%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/passwd_changes.cpp.o
[ 26%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/processes.cpp.o
[ 26%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/routes.cpp.o
[ 27%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/time.cpp.o
[ 28%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/users.cpp.o
[ 28%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/alf.cpp.o
[ 29%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/alf_exceptions.cpp.o
[ 30%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/alf_explicit_auths.cpp.o
[ 30%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/alf_services.cpp.o
[ 31%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/apps.cpp.o
[ 31%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/ca_certs.cpp.o
[ 32%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/interface_addresses.cpp.o
[ 33%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/interface_details.cpp.o
[ 33%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/kextstat.cpp.o
[ 34%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/launchd.cpp.o
[ 35%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/listening_ports.cpp.o
[ 35%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/nvram.cpp.o
[ 36%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/osx_version.cpp.o
[ 37%] Building CXX object osquery/CMakeFiles/osquery_generated_tables.dir/__/generated/tables/startup_items.cpp.o
[ 53%] Built target osquery_generated_tables
Scanning dependencies of target osquery_core
[ 54%] Building CXX object osquery/core/CMakeFiles/osquery_core.dir/sql.cpp.o
[ 55%] Building CXX object osquery/core/CMakeFiles/osquery_core.dir/sqlite_util.cpp.o
[ 56%] Building CXX object osquery/core/CMakeFiles/osquery_core.dir/darwin/test_util.cpp.o
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:13:27: warning: missing terminating '"' character [-Winvalid-pp-token]
  std::string content = R "(
                          ^
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:13:25: error: use of undeclared identifier 'R'
  std::string content = R "(
                        ^
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:35:72: warning: missing terminating '"' character [-Winvalid-pp-token]
                    Wait< / key><false / >< / dict>< / dict>< / plist>)";
                                                                       ^
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:40:27: warning: missing terminating '"' character [-Winvalid-pp-token]
  std::string content = R "(
                          ^
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:40:25: error: use of undeclared identifier 'R'
  std::string content = R "(
                        ^
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:103:33: warning: missing terminating '"' character [-Winvalid-pp-token]
            < / dict>< / plist>)";
                                ^
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:115:27: warning: missing terminating '"' character [-Winvalid-pp-token]
  std::string content = R "(
                          ^
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:115:25: error: use of undeclared identifier 'R'
  std::string content = R "(
                        ^
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:161:47: warning: missing terminating '"' character [-Winvalid-pp-token]
            YES< / string>< / dict>< / plist>)";
                                              ^
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:166:27: warning: missing terminating '"' character [-Winvalid-pp-token]
  std::string content = R "(
                          ^
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:166:25: error: use of undeclared identifier 'R'
  std::string content = R "(
                        ^
/Users/zwass/osquery/osquery/core/darwin/test_util.cpp:190:55: warning: missing terminating '"' character [-Winvalid-pp-token]
            Interactive< / string>< / dict>< / plist>)";
                                                      ^
8 warnings and 4 errors generated.
make[3]: *** [osquery/core/CMakeFiles/osquery_core.dir/darwin/test_util.cpp.o] Error 1
make[2]: *** [osquery/core/CMakeFiles/osquery_core.dir/all] Error 2
make[1]: *** [all] Error 2
make: *** [all] Error 2
```

--- a/.clang-format
+++ b/.clang-format
@@ -19,7 +19,6 @@ IndentCaseLabels: false
 MaxEmptyLinesToKeep: 1
 NamespaceIndentation: None
 ObjCSpaceBeforeProtocolList: false
-PenaltyBreakBeforeFirstCallParameter: 10
 PenaltyBreakComment: 60
 PenaltyBreakString: 1000
 PenaltyBreakFirstLessLess: 20
@@ -39,5 +38,4 @@ SpaceInEmptyParentheses: false
 SpacesInCStyleCastParentheses: false
 SpaceAfterControlStatementKeyword: true
 SpaceBeforeAssignmentOperators: true
-ContinuationIndentWidth: 4
 ...

